xcdat/dataset.py

```diff
@@ -403,7 +410,11 @@ def _has_cf_compliant_time(
         return None
 
     time = ds.cf["T"]
-    units = _split_time_units_attr(time.attrs.get("units"))[0]
+    # if the time units attr cannot be split it is not cf_compliant
+    try:
+        units = _split_time_units_attr(time.attrs.get("units"))[0]
+    except ValueError:
+        return False
```

Comment:
My read is that if there is no "since" the time axis is not cf-compliant [here](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.7/build/ch04s04.html).